code/config.ss

```diff
@@ -1,151 +1,151 @@
-      (unlocked-by)
+      (unlocked_by)
       (difficulty . 1)
       (topics list))
-     
+
      ((slug . leap)
       (uuid . "b94548df-3b35-4ce1-80c7-36179d2b3b86")
       (core . #f)
-      (unlocked-by)
+      (unlocked_by)
       (difficulty . 1)
       (topics))
-     
+
      ((slug . grains)
       (uuid . "397d9cfc-43f9-4c57-bc55-5c94ae8f532e")
       (core . #f)
-      (unlocked-by)
+      (unlocked_by)
       (difficulty . 1)
       (topics math))
-     
+
      ((slug . bob)
       (uuid . "4cf6697c-ad9e-426c-a2d8-6c3695812421")
       (core . #f)
-      (unlocked-by)
+      (unlocked_by)
       (difficulty . 1)
       (topics string))
-     
+
      ((slug . raindrops)
       (uuid . "9f13b913-a650-4cbf-a392-4b5614e1aa2a")
       (core . #f)
-      (unlocked-by)
+      (unlocked_by)
       (difficulty . 1)
       (topics))
-     
+
      ((slug . rna-transcription)
       (uuid . "26bf9c24-ba92-473e-9270-8964f1bf71f2")
       (core . #f)
-      (unlocked-by)
+      (unlocked_by)
       (difficulty . 1)
       (topics string))
-     
+
      ((slug . robot-name)
       (uuid . "62a5a71c-a9eb-416b-809c-c862a4828e8b")
       (core . #f)
-      (unlocked-by)
+      (unlocked_by)
       (difficulty . 1)
       (topics))
-     
+
      ((slug . phone-number)
       (uuid . "8ed74044-2361-459e-aca5-abd73ed5c1cb")
       (core . #f)
-      (unlocked-by)
+      (unlocked_by)
       (difficulty . 1)
       (topics string))
-     
+
      ((slug . anagram)
       (uuid . "c550055c-4bd5-478b-8e91-94347a314aef")
       (core . #f)
-      (unlocked-by)
+      (unlocked_by)
       (difficulty . 1)
       (topics string))
-     
+
      ((slug . nucleotide-count)
       (uuid . "d72cf2c3-9f30-4df1-a69b-04c638bb2426")
       (core . #f)
-      (unlocked-by)
+      (unlocked_by)
       (difficulty . 1)
       (topics))
-     
+
      ((slug . difference-of-squares)
       (uuid . "a3c7a4d2-ba08-47c9-a88e-2311ad02d945")
       (core . #f)
-      (unlocked-by)
+      (unlocked_by)
       (difficulty . 1)
       (topics math))
-     
+
      ((slug . list-ops)
       (uuid . "903374d2-8155-4fe3-bf90-5b6359c7b5cc")
       (core . #f)
-      (unlocked-by)
+      (unlocked_by)
       (difficulty . 1)
       (topics))
-     
+
      ((slug . scrabble-score)
       (uuid . "5b44ccd4-b1f4-4b86-89c8-f1dec5905ccb")
       (core . #f)
-      (unlocked-by)
+      (unlocked_by)
       (difficulty . 1)
       (topics))
-     
+
      ((slug . word-count)
       (uuid . "9460b65d-dc80-4a95-8782-b395d2cc979e")
       (core . #f)
-      (unlocked-by)
+      (unlocked_by)
       (difficulty . 1)
       (topics))
-     
+
      ((slug . two-fer)
       (uuid . "3ecc2d1c-55e0-45c9-ba35-57d7d8cfd51e")
       (core . #f)
-      (unlocked-by)
+      (unlocked_by)
       (difficulty . 1)
       (topics))
 
      ;; math problems
      ((slug . sieve)
       (uuid . "732e4c98-675d-4b67-b672-37ef9c848733")
       (core . #f)
-      (unlocked-by)
+      (unlocked_by)
```

Comment:
It would be better to keep scheme-y names here and make it the responsibility of the pretty printer to convert `-` to `_` imo

Reply:
Okay, so just on the keys?

Can we keep the underscores in the topics list?

Reply:
Currently, the output json has `-` but in the final pass `configlet fmt` modifies them to `_`. I'd prefer to match that behavior, especially since scheme and underscores is a faux pas:

[https://mumble.net/~campbell/scheme/style.txt](https://mumble.net/~campbell/scheme/style.txt)

Reply:
Speaking of which, we should add that to resources.md

Reply:
Since the topics are keys and not string it might be simpler to implement than I was thinking.